src/assets/htmx.md

```diff
@@ -0,0 +1,347 @@
+---
+title: htmx
+authorName: Niraj Acharya
+authorAvatar: https://avatars.githubusercontent.com/u/122071597?v=4
+authorLink: https://github.com/nirajacharya2
+createdAt: Sep 29, 2023
+tags: js, html, go, htmx
+banner: https://raw.githubusercontent.com/JankariTech/blog/master/src/imgs/fallback_banner.png
+---
+
+Htmx is ui library written in JavaScript that extends the features of html. It can be extended to make AJAX requests, do CSS transitions, connect to WebSockets and handle Server Sent Events inside a html document with just some delacritive html attributes.
+
+Htmx was created by Carson Gross in 2013 because he found html to be limited for what he was trying to achieve. Those limitations of html htmx tries to solve are:
```

Comment:
the last sentence is a bit difficult to understand

```suggestion
Htmx was created by Carson Gross in 2013 because he found html to be limited for what he was trying to achieve. Those limitations of html, which htmx tries to solve, are:
```
or

```suggestion
Htmx was created by Carson Gross in 2013 because he found html to be limited for what he was trying to achieve.  htmx tries to solve the following limitations of html:
```